modules/testing/src/views/lite/scenarios/ScenarioDetails.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+
+import { Button, Collapse, FormGroup, Icon, InputGroup, Intent, TextArea } from '@blueprintjs/core'
+
+export default props => {
```

Comment:
would that be possible to share the ScenarioDetails component between like lite and the one we use in fullview ?